bin/start

```diff
@@ -54,18 +58,22 @@ done
 
 export RUN_IN_DOCKER
 export CONJUR_DEPLOYMENT
-
-source test/bootstrap.env
+export DEV_ENV
 
 # summon environment variable
 export CONJUR_MAJOR_VERSION=4
 
+source bootstrap.env
 cd test
+
 if [[ $DEMO = true ]]; then
   runScriptWithSummon ./run_demo.sh
```

Comment:
i am guessing the demo will be removed as part of this effort. am i correct?

Reply:
Do we want to remove all traces of the demo here? Or leave it as is for us to build on later?

Reply:
why do we need a demo if we have a dev env?